vignettes/decorate-multiple-modules-outputs.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Decorate modules output"
+author: "NEST CoreDev"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Decorate modules output}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
```

Comment:
This vignette could be a part of the other `decorate-module-output` vignette. We could have just one